appIos/appIos.xcodeproj/project.pbxproj

```diff
@@ -324,7 +324,7 @@
 				ENABLE_PREVIEWS = YES;
 				FRAMEWORK_SEARCH_PATHS = "";
 				INFOPLIST_FILE = appIos/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
Dammit Apple… I really don’t want to bump the deployment target just for this. Might look for alternatives before doing this. Since this is a template it means all apps made from this will require iOS 15+. Not the end of the world but definitely annoying

Reply:
Alternative [here](https://www.appcoda.com/swiftui-search-bar/) but still unsure of whether we should do our own or not. 

Compose doesn’t have one out of the box so it’ll need a custom implementation too, so we could have the two search bars implemented locally with custom views.

What do you think about this one? Code seems simple enough to copy 😂

Reply:
Right... it does seem achievable even. I do wonder if given it's a template should target latest version? this is iOS and I think people are usually in the latest

as for learning I'm all-in with a bit of lower level, so if you want we can implement what's in that article!

Reply:
Yeah I'm thinking the same - might as well target the latest version since we don't have to cater for backwards compatibility and should be using latest tech. Lets do that 🧑‍⚖️

Could we then update the iOS version everywhere? You should use Android Studio's search as it's much more detailed than Xcode and it will show you all places where we have iOS 14.0 set - all those should be 15.0 now. 

<img width="626" alt="Screenshot 2022-04-21 at 09 14 17" src="https://user-images.githubusercontent.com/7644787/164396159-138c0e44-f257-4968-95af-a8296acb0b7d.png">

Reply:
I actually started building the view from the example, I'm fine anyways. Probably staying in iOS 15 makes sense anyways. I'll branch off the 'custom' version separately

Reply:
Pushed the target change to 15 gogogo

Reply:
I think eventually we’ll move to the new way and compose will probably add something similar, so may as well have it now and increase the min version. 

Custom version probably won’t have as many features as the Apple one too.